inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/model/definition/ServiceTaskListenerProvider.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.manager.common.model.definition;
+
+import org.apache.inlong.manager.common.event.task.TaskEventListener;
+import org.apache.inlong.manager.common.model.WorkflowContext;
+
+/**
+ * An object capable of providing instances of TaskEventListener
+ */
+public interface ServiceTaskListenerProvider<T extends TaskEventListener> {
+
+    public Iterable<T> get(WorkflowContext workflowContext, ServiceTaskType serviceTaskType);
```

Comment:
Modifier 'public' is redundant for interface methods.

Reply:
done